devchat.go

```diff
@@ -437,19 +491,37 @@ func (u *user) pickUsername(possibleName string) {
 		possibleName = cleanName(possibleName)
 	}
 	u.name = possibleName
-	u.changeColor(*colorArr[rand.Intn(len(colorArr))]) // also sets prompt
+	var colorIndex = rand.Intn(len(colorstyles))
+	u.changeColor(colorstyles[colorIndex].name) // also sets prompt
+}
+
+// Applies color from name
+func (u *user) changeColor(colorName string) {
+	color := getColor(colorName)
+	u.changeColorstyle(color)
 }
 
-func (u *user) changeColor(color color.Color) {
-	u.name = color.Sprint(stripansi.Strip(u.name))
-	u.color = color
+// Applies color from colorstyle
+func (u *user) changeColorstyle(color colorstyle) {
```

Comment:
Merge this function into `changeColor` directly (we're trying to make the entire outward API work with colors as named strings)

Reply:
For some reason I'm not able to make this a nice little clickable suggestion